legal-api/src/legal_api/services/authz.py

```diff
@@ -71,9 +104,10 @@ def authorized(  # pylint: disable=too-many-return-statements
             or jwt.validate_roles([COLIN_SVC_ROLE]):
         return True
 
-    # allow IDIM view access on everything
-    if len(action) == 1 and action[0] == 'view' and jwt.validate_roles([ACCOUNT_IDENTITY]):
-        return True
+    # allow IDIM and Competent Authorities view access on everything
+    if len(action) == 1 and action[0] == 'view':
```

Comment:
Just a note
You could have put 2nd if expression in the bracket and join it with `and`, if there will be no granularity needed for some other options.

Reply:
yeah I just separated them for readability